server/controllers/area.controllers.js

```diff
@@ -31,7 +32,10 @@ const AreaController = {
 
   getAreas(req, res, next) {
     const { state } = req.params;
-    getAreas(state)
+    const adminOrEditorId = req.headers.authorization
```

Comment:
I appreciate your thoroughness here, however, since the area information doesn't have any sensitive information, we can keep this simple to check if only the authorization token exist only:

```suggestion
    const tokenIsPresent = !!req.headers.authorization;
```